lib/node_modules/@stdlib/iter/cusome-by/docs/repl.txt

```diff
@@ -0,0 +1,49 @@
+The predicate function is provided two arguments:
```

Comment:
This file does not adhere to project conventions. See the REPL text guide in `/docs`.